addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1702,7 +1702,7 @@ const Wysiwyg = Widget.extend({
         const selectedTds = this.odooEditor.document.querySelectorAll('td.o_selected_td');
         const applyOpacity = !selectedTds.length && eventName === 'backColor' && !this.colorpickers['backColor'].sections['theme-colors'].classList.contains('d-none');
         // Applying 60% opacity to the selected common color by default
-        // to make text highlighting more useable.
+        // to make text highlighting more enhanced in dark mode.
```

Comment:
Drop this change since I rewrote the comment in the suggestion in my other comment.